openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -164,7 +170,7 @@ def get_all_object_tags(
     all_object_tags = ObjectTag.objects.filter(
         Q(tag__isnull=False, tag__taxonomy__isnull=False),
         object_id_clause,
-    ).select_related("tag__taxonomy")
+    ).select_related("tag__taxonomy").order_by("object_id")
```

Comment:
Good catch @ChrisChV! Thank you for fixing that.

Reply:
Thanks!